frontend/components/Company/ListTabs.vue

```diff
@@ -0,0 +1,109 @@
+
+<script setup>
+import { computed } from 'vue'
+import { TabGroup, TabList, Tab, TabPanels, TabPanel } from '@headlessui/vue'
+import { useBuildLists } from '@/composables/useBuildLists.js'
+import FacebookIcon from '@/components/Icons/FacebookIcon.vue'
+import LinkedinIcon from '@/components/Icons/LinkedinIcon.vue'
+import TwitterIcon from '@/components/Icons/TwitterIcon.vue'
+import YouTubeIcon from '@/components/Icons/YouTubeIcon.vue'
+import SlideShareIcon from '@/components/Icons/SlideShareIcon.vue'
+
+const props = defineProps({
+  name: {
+    type: String,
+    default: '',
+  },
+  meetups: {
+    type: Array,
+    default: () => [],
+  },
+})
+
+const buildLists = computed(() => {
+  if (props.meetups.length === 0) return {}
+  const { lists } = useBuildLists(props.meetups, props.name)
+  return lists.value
+})
+
+</script>
+
+<template>
+  <div class="my-10 bg-white text-center text-lg font-bold tracking-tight text-zinc-700 shadow-lg md:text-2xl">
+    <div class="lg:py-18 mx-auto block w-full max-w-7xl p-4 sm:px-6 lg:px-8">
+      <h2 class="first-letter:text-laravel lg:py-18 w-full justify-center border-b py-6 px-7 text-left text-2xl font-bold sm:px-6 md:text-4xl lg:px-8"> 
+        Meetup'owe statystyki
```

Comment:
I'm not sure if "statistics" here is a good word.

Reply:
maybe "Udział w Meetup'ach" or "Meetup'owy wkład"?

Reply:
Meetupowy wkład will be okay.